pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -736,6 +736,16 @@ type VerticalPodAutoscaler struct {
 	// (default: 24h)
 	// +optional
 	MemoryHistogramDecayHalfLife *metav1.Duration `json:"memoryHistogramDecayHalfLife,omitempty" protobuf:"bytes,16,opt,name=memoryHistogramDecayHalfLife"`
+	// MemoryAggregationInterval is the length of a single interval, for which the peak memory usage is computed.
+	// (default: 24h)
+	// +optional
+	MemoryAggregationInterval *metav1.Duration `json:"memoryAggregationInterval,omitempty" protobuf:"bytes,17,opt,name=memoryAggregationInterval"`
+	// MemoryAggregationIntervalCount is the number of consecutive memory-aggregation-intervals which make up the
+	// MemoryAggregationWindowLength which in turn is the period for memory usage aggregation by VPA. In other words,
+	// `MemoryAggregationWindowLength = memory-aggregation-interval * memory-aggregation-interval-count`.
+	// (default: 8)
+	// +optional
+	MemoryAggregationIntervalCount *int64 `json:"memoryAggregationIntervalCount,omitempty" protobuf:"fixed64,18,opt,name=memoryAggregationIntervalCount"`
```

Comment:
Protobuf generated automatically. 
```suggestion
	MemoryAggregationIntervalCount *int64 `json:"memoryAggregationIntervalCount,omitempty" protobuf:"varint,18,opt,name=memoryAggregationIntervalCount"`
```

Reply:
done.